src/pages/SearchPage.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import CropItem from '../components/CropItem'
+import Header from '../components/Header'
+
+SearchPage.propTypes = {
+  crops: PropTypes.array.isRequired,
+  onClickDetails: PropTypes.func.isRequired,
+  onClickFavorites: PropTypes.func.isRequired,
+}
+
+export default function SearchPage({
+  crops,
+  onClickFavorites,
+  onClickDetails,
+}) {
+  const [searchTerm, setSearchTerm] = useState('')
+  const [searchResults, setSearchResults] = useState([])
```

Comment:
you could think about putting this in a hook but since those two lines don't mess up your code it is probably unnecessary